src/txkube/test/test_authentication.py

```diff
@@ -29,11 +29,15 @@
 from cryptography.hazmat.backends import default_backend
 
 from twisted.python.filepath import FilePath
+from twisted.internet.address import IPv4Address
+from twisted.internet.interfaces import IReactorPluggableNameResolver
 from twisted.internet.protocol import Factory
 from twisted.web.http_headers import Headers
 from twisted.test.iosim import ConnectionCompleter
 from twisted.test.proto_helpers import AccumulatingProtocol, MemoryReactorClock
 
+from zope.interface import implementer
```

Comment:
Can you move this up above the Twisted imports?  (because Twisted depends on zope.interface)